conanfile.py

```diff
@@ -116,6 +117,7 @@ def requirements(self):
         'external/*',
         'include/*',
         'src/*',
+        'tests/*',
```

Comment:
This line is no longer needed if you're using `src/tests` instead of `tests`. I still recommend `tests` as a matter of convention.